docs/partials/_elemental-operator-install.md

```diff
@@ -9,11 +9,17 @@ We will use the Helm package manager to install the elemental-operator chart int
 The Elemental Operator chart is distributed via an OCI registry: Helm correctly supports OCI based registries starting from the v3.8.0 release.
 :::
 
+:::warning Swap charts installation order when upgrading from elemental-operator release < 1.2.4
+When upgrading from an elemental-operator release embedding the Elemental CRDs (version < 1.2.4) the elemental-operator-crds chart installation will fail.
+You will need to upgrade the elemental-operator chart first, and only then install the elemental-operator-crds chart.
+:::
+
 <Tabs>
-<TabItem value="stableOperator" label="Stable version (x86-64, ARM64 (Raspberry Pi 4))" default>
+<TabItem value="stagingOperator" label="Staging version (x86-64, ARM64 (Raspberry Pi 4))" default>
```

Comment:
We should remember to switch this back to stable once we finalize the release channels and the split charts land on stable as well. I think it will be safer for new users. 
For the moment however I think this is ok.

Reply:
in order to move it to Stable, there is like  a "release" procedure that copies from Next (the docs folder).
So, yep, the plan is have this on Next, do the "release process" and update the links in Stable.
The links in Stable should be updated anyway with IBS links.